src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -878,19 +889,44 @@ const getConfigSecretDef = (frameworkName, secrets) => {
 const list = async (
   attributes,
   filters,
+  tagsFilter,
   order,
   offset,
   limit,
   withTotalCount,
 ) => {
   let frameworks;
   let totalCount;
+
+  if (Object.keys(tagsFilter).length !== 0) {
+    const frameworkNames = await databaseModel.Tag.findAll({
+      attributes: ['frameworkName'],
```

Comment:
add other filter and order & limit here?

Reply:
> add other filter and order & limit & offset here? if there are many framework with this tag, querying them will cause memory/CPU issues.

Merged these two sql sentences into one to avoid data transfer between DB and RestServer.